reports/historical/historical.py

```diff
@@ -88,9 +88,12 @@ def import_from_file(self, fname, coverage, region=None, country=None, year=None
             )
 
     def historical_sheets(self, coverage, year):
+        suppressed_sheets = ['42', '45', '53', '54', '66', '67', '68', '71', '72', '73', '74', '75', '76', '77', '78']
         sheets = []
         for sheets_by_year in WS_INFO.values():
             sheet = sheets_by_year.get(year)
-            if sheet and ("historical" in sheet and coverage in sheet["reports"]):
+            if sheet \
+               and ("historical" in sheet and coverage in sheet["reports"])\
+               and sheet["historical"] not in suppressed_sheets:
```

Comment:
This would work but I don't think it's the right approach. My understanding is `reports` array list in the `report_details.py` controls when a sheet should be imported / report generated for it. All we have to do is:

1. Complete suppression: Remove all types i.e. `'reports': []`,
2. Region suppression: Remove `region` from the `reports` array list, and
3. Country suppression: remove `country` from the `reports` array list.